packages/components/organisms/f-cookie-banner/package.json

```diff
@@ -61,7 +61,7 @@
     "@justeat/f-button": "4.x",
     "@justeat/f-link": "3.x",
     "@justeat/f-mega-modal": "7.x",
-    "@justeat/f-vue-icons": "2.x",
+    "@justeat/f-vue-icons": "3.x",
```

Comment:
An icon got renamed in `3.x`, however this component doesn't use it, so it will continue to work as before.